spec/models/book_spec.rb

```diff
@@ -2,4 +2,6 @@
 
 RSpec.describe Book, type: :model do
   it { is_expected.to validate_presence_of(:title) }
+  it { is_expected.to have_many(:book_lists) }
+  it { is_expected.to have_many(:lists).through(:book_lists) }
 end
```

Comment:
I was reading through some of these examples but was not sure if we need to add a check for foreign keys on the join table.

https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_record/association_matcher.rb#L373